pages/api_integration-full-node-streaming.md

```diff
@@ -342,6 +518,10 @@ By protocol design, each validator has their own version of the orderbook and op
 
 Note that DeliverTx maps to exec mode `execModeFinalize`.
 
+### Finalized Subaccount Updates
+
+Only finalized subaccount updates are sent.
```

Comment:
more info probably better to explain this why we only see exec mode 102 and 7.

Reply:
done